<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kendali Motor Arduino</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_turn_forward_left">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>90</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn Forward Left</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_LogMessage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="button_serial">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_exit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_turn_forward_right">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn Forward Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_forward">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reverse">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_stay">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_turn_reverse_left">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>130</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn Reverse Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_turn_reverse_right">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn Reverse Right</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
